npm/ng-packs/package.json

```diff
@@ -44,19 +44,6 @@
   },
   "private": true,
   "devDependencies": {
-    "@abp/ng.account": "~7.4.0-rc.5",
-    "@abp/ng.account.core": "~7.4.0-rc.5",
-    "@abp/ng.core": "~7.4.0-rc.5",
-    "@abp/ng.feature-management": "~7.4.0-rc.5",
-    "@abp/ng.identity": "~7.4.0-rc.5",
-    "@abp/ng.oauth": "~7.4.0-rc.5",
-    "@abp/ng.permission-management": "~7.4.0-rc.5",
-    "@abp/ng.schematics": "~7.4.0-rc.5",
-    "@abp/ng.setting-management": "~7.4.0-rc.5",
-    "@abp/ng.tenant-management": "~7.4.0-rc.5",
-    "@abp/ng.theme.basic": "~7.4.0-rc.5",
-    "@abp/ng.theme.lepton-x": "^2.4.0-rc.4",
-    "@abp/ng.theme.shared": "~7.4.0-rc.5",
     "@abp/utils": "~7.4.0-rc.5",
```

Comment:
`"@abp/ng.theme.lepton-x": "^2.4.0-rc.5"` is missing. There is not local reference path for this package!